### PR TITLE
PAAS-1855 Create Augmented Search install package

### PR DESCRIPTION
**Pull request by laurentfufu**
JIRA issue: https://jira.jahia.org/browse/PAAS-1855

Short description:

### CODE REVIEW

**Review by kevinvandel — DISMISSED** [review]

File: packages/jahia/augmented-search-install.yml [hunk 1]
```diff
@@ -58,13 +59,26 @@ actions:
...
         - if(globals.jahia810):
             - setGlobals:  # if jahia 8.1.0
-                graphql_ver: 2.8.0
+                graphql_ver: 2.8.1
+        - isVersionStrictlyLower:
+            a: ${globals.jahiaVersion}
+            b: 8.1.2.0
+            res: lessjahia812
+        - if(globals.lessjahia812):
+            - setGlobals:  # if jahia < 8.1.2 jContent have to be upgraded
```
Comment (kevinvandel) [inline packages/jahia/augmented-search-install.yml:68]: *has :grin:

File: packages/jahia/augmented-search-install.yml [hunk 2]
```diff
@@ -58,13 +59,26 @@ actions:
...
     - if(globals.jahia810):  # if jahia 8.1.0 then graphql dxm provider must be upgraded
+        - disableHaproxyHealthcheck  # upgrading graphql temporarily break the healthcheck
         - installOrUpgradeModule:
             moduleSymname: graphql-dxm-provider
             moduleVersion: ${globals.graphql_ver}
+        - enableHaproxyHealthcheck
+    - if(globals.lessjahia812):
+        - installOrUpgradeModule:
+            moduleSymname: jcontent
+            moduleVersion: ${globals.jcontent_ver}
```
Comment (kevinvandel) [inline packages/jahia/augmented-search-install.yml:81]: Is it really useful to define a `jcontent_ver` global ? Couldn't we just put 2.7.0 here ?

File: packages/jahia/augmented-search-install.yml [hunk 3]
```diff
@@ -76,8 +90,16 @@ actions:
         moduleVersion: ${globals.augsearch_ver}
 
   setNodeGroupData:
+    - cmd[${nodes.cp.first.id}]: |-
+        source /metadata_from_HOST
+        if [ -z $JEL_ENV_ROLE ]; then
+          echo "JEL_ENV_ROLE variable is not defined in /metadata_from_HOST"
+          exit 1
+        fi
+        echo $JEL_ENV_ROLE
```
Comment (kevinvandel) [inline packages/jahia/augmented-search-install.yml:99]: Isn't it already done in a previous PR by Axel ?

**Review by kevinvandel — CHANGES_REQUESTED** [review]

**Review by kevinvandel — APPROVED** [review]

File: packages/jahia/augmented-search-install.yml (hunk 2 above)
Comment (laurentfufu) [reply packages/jahia/augmented-search-install.yml:81]: all module versions are defined in `determineModulesVersions`, I think it's better that way even if indeed here, the global `jcontent_ver` is only set up and used once

**Pull request merged**